toqito/channel_metrics/fidelity_of_separability.py

```diff
@@ -0,0 +1,198 @@
+"""Add functions for channel fidelity of Separability as defined in [Phil23]_.
+
+The constrainsts for this function are positive partial transpose(PPT)
+& k-extendible channels.
+"""
+
+from toqito.state_props import is_pure
+from toqito.matrix_props import is_density
+import picos
+import numpy as np
+from toqito.perms import symmetric_projection
+from toqito.perms import permute_systems
+
+
+
+def fidelity_of_separability(psi: np.ndarray, psi_dims: list[int], k: int = 1) -> float:
+    r"""
+    Define the first benchmark introduced in Appendix I of [Phil23]_.
+
+    If you would like to instead use the benchmark introduced in Appendix H,
+    go to :obj:`toqito.state_metrics.fidelity_of_separability`.
+
+    In [Phil23]_ a variational quantum algorithm (VQA) is introduced to test
+    the separability of a general bipartite state. The algorithm utilizes
+    quantum steering between two separated systems such that the separability
+    of the state is quantified.
+
+    Due to the limitations of currently available quantum computers, two
+    optimization semi-definite programs (SDP) benchmarks were introduced to
```

Comment:
Nitpick: Your preference, but throughout the rest of `toqito`, the term "semidefinite" is preferred over "semi-definite".